### PR TITLE
Upgrade Redwood

### LINKED ISSUE #38: demo doesn't work
https://redwood-example-invoice.netlify.app/#

1. signup
2. click on the link by email
3. app says logged in

"Try it out" doesn't work (does a POST graphql with 500 response) 
"Get started" says Logged in as test

### PR DESCRIPTION
**Pull request by dale-french**
Fixes #38 

* Upgraded Redwood to `0.19.3`
* Upgraded Tailwind

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 .env
 .netlify
+.redwood
 dev.db
 dist
 dist-babel
 node_modules
 yarn-error.log
-.env
 web/public/mockServiceWorker.js
```
Comment (thedavidprice) [inline .gitignore:11]:
```suggestion
web/public/mockServiceWorker.js

# Deployment	
.serverless
```

We should probably keep this. Peter's been testing the Serverless Framework deploy with this project.

**Review by thedavidprice — CHANGES_REQUESTED** [review]
Looking good so far. Check out the "Manual Code Modifications" for [v0.15.0](https://github.com/redwoodjs/redwood/releases/tag/v0.15.0). I'd like to see the update for `schema.prisma` binaryTargets in this upgrade.

And do go ahead and drop IE 11 targets from browserlist. Example here: https://github.com/redwoodjs/create-redwood-app/commit/51c39087f8c7b7037a26ec015e4e140c3997a38a

**Comment by dale-french** [comment]
Cool stuff - I have added in the changes requested.

I have also updated the Netlify build command to match https://github.com/redwoodjs/create-redwood-app/pull/87/files

**Review by thedavidprice — APPROVED** [review]
🚀

**Comment by thedavidprice** [comment]
Looking good! Merging now — expecting this to fail due to current Node v12.19 + Prisma bug. Will take steps accordingly.

**Comment by thedavidprice** [comment]
And there's the build error:
```bash
Error: Schema Parsing P1012
2:20:22 PM: 
2:20:22 PM: Get config 
2:20:22 PM: error: Error validating datasource `sqlite`: The URL for datasource `sqlite` must start with the protocol `file:`.
2:20:22 PM:   -->  schema.prisma:3
2:20:22 PM:    | 
2:20:22 PM:  2 |   provider = "sqlite"
2:20:22 PM:  3 |   url      = env("DB_HOST")
2:20:22 PM:    | 
2:20:22 PM: 
2:20:22 PM: Validation Error Count: 1
2:20:22 PM: error Command failed with exit code 1.
2:20:22 PM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2:20:22 PM: Migrate database up... [failed]
2:20:22 PM: → Command failed with exit code 1: yarn prisma migrate up --experimental --create-db --auto-approve
2:20:22 PM: Command failed with exit code 1: yarn prisma migrate up --experimental --create-db --auto-approve
```

**Comment by thedavidprice** [comment]
@dale-french This is all progress. Just know what you see in production != to the current status in `main`.

Ah, I forgot about this new Prisma schema syntax https://github.com/redwoodjs/example-invoice/commit/b42abb88e39b59f342b129f9385f36899357590f

And made the temporary Node version change — created this Issue as a future reminder to revert: https://github.com/redwoodjs/example-invoice/issues/42

 BUT, now production DB migrations are out of sync, which is something I'll need Peter to help with:

```
2:36:53 PM: Migrate database up... [started]
2:36:53 PM: $ /opt/build/repo/node_modules/.bin/prisma migrate up --experimental --create-db --auto-approve
2:36:55 PM: Error: Local and remote migrations are not in lockstep. We have migration 20200730175800-migration locally and 20200227093425 remotely at the same position in the history.
2:36:55 PM: error Command failed with exit code 1.
2:36:55 PM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2:36:56 PM: Migrate database up... [failed]
2:36:56 PM: → Command failed with exit code 1: yarn prisma migrate up --experimental --create-db --auto-approve
2:36:56 PM: Command failed with exit code 1: yarn prisma migrate up --experimental --create-db --auto-approve
```

**Pull request merged**